kimnyamhondera/blink182/blink182.ino

```diff
@@ -0,0 +1,18 @@
+/* This sketch is sketch for sketch this sketch task sketch https://trello.com/c/ba783WmU/2-blink-an-led */
+
+int pin=13; /* declare pin */
```

Comment:
You've specified the pin to be the value '13' so in the rest of your code where you have '13' you can just replace it with the word 'pin'